new-site/docs-site/content/0.19.0/guide/typesense-dynamodb.md

```diff
@@ -0,0 +1,297 @@
+# DynamoDB Integration with Typesense
+
+Hey there! This post give you a guide to integrate Typesense custer with AWS DynamoDB by setting up a trigger with AWS Lambda.
+
+![Typesense DynamoDB Integration Chart](~@guide-images/typesense-dynamodb/dynamodb-typesense-integration-1.png)
+
+## Create AWS DynamboDB with streams enabled
+
+Create a DynamoDB with your choice of name and partition key ("id" is recommended). Now, after creating a DynamoDB database you should enable streams in the "Overview" section.
+
+![DynamoDB streams](~@guide-images/typesense-dynamodb/dynamodb-typesense-integration-2.png)
+
+If you are doing this in AWS CLI,
+
+```bash
+aws dynamodb create-table \
+    --table-name typensense \
+    --attribute-definitions AttributeName=id,AttributeType=N \
+    --key-schema AttributeName=id,KeyType=HASH  \
+    --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 \
+    --stream-specification StreamEnabled=true,StreamViewType=NEW_AND_OLD_IMAGES
+```
+## Setting up Triggers for DyanmoDB
```

Comment:
Could you add Step number, and use an action verb. 

For eg: `Step 2:  Set up a trigger`